.editorconfig

```diff
@@ -0,0 +1,9 @@
+root = true
```

Comment:
If we decide to want to keep this, I can complete the settings for all other file types etc. (I believe VSCode can generate this based on the file structure)